client/reducers/allProducts.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+// ACTION TYPES
```

Comment:
Kim and I think that "getAllProducts" and "GET_ALL_PRODUCTS" would be more clear than setAllProducts, etc.